documentation/self-host/community-edition/install-and-build.mdx

```diff
@@ -84,19 +101,26 @@ Let's understand the major environment variables:
 4. `REFRESH_TOKEN_VALIDITY`: Validity of the refresh token for auth (in ms).
 5. `ACCESS_TOKEN_VALIDITY`: Validity of the access token for auth (in ms).
 6. `JWT_SECRET`, `SESSION_SECRET`: Secret Keys for security purposes.
-7. `REDIRECT_URL`: This is a fallback URL to debug when the actual redirects fail.
-8. `WHITELISTED_ORIGINS`: URLs of Hoppscotch backend, admin dashboard, and the frontend app.
-9. `VITE_ALLOWED_AUTH_PROVIDERS`: Allows you to specify which auth providers you want to enable.
-10. `MAILER_SMTP_URL`: The SMTP URL for email delivery.
-11. `MAILER_ADDRESS_FROM`: The email address that you would be using.
-12. `RATE_LIMIT_TTL`: The time it takes to refresh the maximum number of requests being received.
-13. `RATE_LIMIT_MAX`: The maximum number of requests that Hoppscotch can handle under `RATE_LIMIT_TTL`.
-14. `VITE_BASE_URL`: This is the URL where your deployment will be accessible from.
-15. `VITE_SHORTCODE_BASE_URL`: A URL to generate shortcodes for sharing, can be the same as `VITE_BASE_URL`.
-16. `VITE_BACKEND_GQL_URL`: The URL for GraphQL within the instance.
-17. `VITE_BACKEND_WS_URL`: The URL for WebSockets within the instance.
-18. `VITE_BACKEND_API_URL`: The URL for REST APIs within the instance.
-19. `VITE_APP_TOS_LINK` and `VITE_APP_PRIVACY_POLICY_LINK` are optional and are used to configure the links to the Terms & Conditions and Privacy Policy.
+7. `ALLOW_SECURE_COOKIES`: Determines if cookies should only be sent over secure (HTTPS) connections. 
```

Comment:
Lets put this here after the PR is merged.

Reply:
Got it.